src/main/resources/hibernate.cfg.xml

```diff
@@ -0,0 +1,14 @@
+<hibernate-configuration>
+    <session-factory>
+        <property name="hibernate.dialect">org.hibernate.dialect.MySQLDialect</property>
+        <property name="connection.url">jdbc:mysql://localhost:3306/movies_app?serverTimeZone=UTC</property>
+        <property name="connection.driver_class">com.mysql.cj.jdbc.Driver</property>
+        <property name="connection.username">root</property>
+        <property name="connection.password">DimaSlava</property>
+        <property name="show_sql">true</property>
+        <property name="hbm2ddl.auto">create-drop</property>
+        <mapping class="mate.academy.hibernate.relations.model.Actor"/>
+        <mapping class="mate.academy.hibernate.relations.model.Country"/>
+        <mapping class="mate.academy.hibernate.relations.model.Movie"/>
+    </session-factory>
+</hibernate-configuration>
```

Comment:
just to split up a bit that wall of code into logical sections + new line at the end of the file
```suggestion
        <property name="hbm2ddl.auto">create-drop</property>
        
        <mapping class="mate.academy.hibernate.relations.model.Actor"/>
        <mapping class="mate.academy.hibernate.relations.model.Country"/>
        <mapping class="mate.academy.hibernate.relations.model.Movie"/>
    </session-factory>
</hibernate-configuration>

```

Reply:
Thanks))